app/assets/javascripts/admin/addon/components/modal/scrub-rejected-user.gjs

```diff
@@ -0,0 +1,63 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { isEmpty } from "@ember/utils";
+import DButton from "discourse/components/d-button";
+import DModal from "discourse/components/d-modal";
+import TextField from "discourse/components/text-field";
+import { i18n } from "discourse-i18n";
+
+export default class ScrubRejectedUserModal extends Component {
+  @tracked isScrubbing = false;
+  @tracked scrubReason = "";
+
+  @action
+  async confirmScrub() {
+    this.isScrubbing = true;
+    await this.args.model.confirmScrub(this.scrubReason);
```

Comment:
Just wondering if errors are handled by the `this.args.model.confirmScrub` function.

Reply:
Only with a `popupAjaxError()`. Generally, the errors that could be returned by the scrub endpoint should only be encountered when someone is deliberately messing with the client-side state to display UI elements that otherwise wouldn't be available, or if the client-side state has gotten substantially out of sync with reality. The `notify_users()` call in `ReviewableUser#scrub` should hopefully keep the latter to a minimum.